crypto/evp/pkey_mac.c

```diff
@@ -377,6 +410,14 @@ static int pkey_mac_ctrl(EVP_PKEY_CTX *ctx, int type, int p1, void *p2)
                                                       p2, p1);
                 params[params_n] = OSSL_PARAM_construct_end();
 
+                if (hctx->ctx == NULL) {
```

Comment:
Does the one on line 377 also need a check?

Reply:
Yes. Added.